fuzz/README.md

```diff
@@ -0,0 +1,110 @@
+# Fuzzing
+
+`simplicity` has a fuzzing harness setup for use with honggfuzz.
+
+To run the fuzz-tests as in CI -- briefly fuzzing every target -- simply
+run
+
+    ./fuzz.sh
+
+in this directory.
+
+You need to install the Rust crate honggfuzz-rs.
+
+To build the crate you must have libunwind on your system, as well as
+libopcodes and libbfd from binutils **2.38** on your system. The most
+recently-released binutils 2.39 has changed their API in a breaking way.
+
+Then you can install it via cargo
+
+    cargo install honggfuzz
+
+On Nix, the installation is automated by using the provided nix shell
+
+    nix-shell
+
+Then run fuzz.sh as above.
+
+# Long-term fuzzing
+
+To see the full list of targets, the most straightforward way is to run
+
+    listTargetNames
```

Comment:
Or you can just run `cargo hfuzz run x` and it will give you the list, and then you don't need to source anything.